### ISSUE TITLE
Bug: DEVMODEA is incorrectly sized and has wrong layout

### ISSUE THREAD

**Issue by TheJelmega** [microsoft/windows-rs#2264]
### Which crate is this about?

windows

### Crate version

0.43.0

### Summary

When trying to query info about a display, I found that the size is of the DEVMODEA is incorrect and that the layout also seems to be wrong. From the tests, the size of DEVMODEA should be 156, but size_of::<DEVMODEA>() returns 164.
The layout is also incorrect, as for example `dmPelsHeight` value is in `dmBitsPerPel`.

By quicly checking the definitions, I found some of the incorrect values:
- dmTTOption: DEVMODE_TRUETYPE_OPTION is 4-bytes, MSDN says 2-bytes (short)
- dmColor: DEVMODE_COLOR is 4-bytes, MSDN says 2-bytes (short)
- dmCollate: DEVMODE_COLLATE is 4-bytes, MSDN says 2-bytes
- Not sure where the last bytes would be, but I guess padding.

### Reproducible example

```rust
use windows::Win32::Graphics::Gdi::DEVMODEA;

fn main() {
    assert_eq!(std::mem::size_of::<DEVMODEA>(), 156);
}
```

### Crate manifest

```toml
[dependencies.windows]
version = "0.43.0"
features = [
    "Win32_Graphics_Gdi",
    "Win32_Foundation"
]
```

### Expected behavior

size_of::<DEVMODEA>() == 156

value of `dmPelsHeight` is located in `dmPelsHeight`

### Actual behavior

size_of::<DEVMODEA>() == 164

value of `dmPelsHeight` is located in `dmBitsPerPel`

**Comment by tim-weis** [comment]
Is this the same issue as #2138?

**Comment by TheJelmega** [comment]
Yeah, it's the same, didn't find that other one, as I was looking for DEVMODEA  in the issue finder, not just DEVMODE
